psi4/driver/procrouting/proc.py

```diff
@@ -1781,9 +1862,8 @@ def run_dfocc_property(name, **kwargs):
     dfocc_wfn = core.dfocc(ref_wfn)
 
     # Shove variables into global space
-    if name in ['mp2', 'omp2']:
-        for k, v in dfocc_wfn.variables().items():
-            core.set_variable(k, v)
+    for k, v in dfocc_wfn.variables().items():
```

Comment:
I forget why this was nested, can you comment on it?

Reply:
@loriab would know for sure, since she wrote this code [a year ago](https://github.com/psi4/psi4/commit/880f87f5b2765ed00efaf6dd63c406b574c40157). My read of the situation was that she was only checking that MP2 worked at the time and wasn't concerned about the rest.

Reply:
Yeah, as discussed at ccqc, this should go back to mp2/omp2 only since other mtds haven't been transitioned from globals to wfn qcvars.